java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -763,6 +763,20 @@ public void functionLoad_and_functionList(boolean withRoute) {
 
         // TODO test with FCALL
 
+        var fdelete =
```

Comment:
Do we want a test for when the library name doesn't exit?

Reply:
Added, thanks